packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -189,7 +190,7 @@ const Services = WebexPlugin.extend({
       formattedQuery = {};
 
       if (queryKey === 'email') {
-        formattedQuery.emailhash = sha256(query.email.toLowerCase()).toString();
+        formattedQuery.emailhash = sha256(query.email?.toLowerCase())?.toString();
```

Comment:
Does our babel/node version support this?

Reply:
indeed it does !

Reply:
Yes!